### ISSUE TITLE
Tab: STRAC > IPA

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1484]
https://dev-onrr-frontend.app.cloud.gov/compliance-enforcement/cooperative
https://dev-onrr-cms.app.cloud.gov/admin/content/pages/28
https://qmcq5k.axshare.com/#id=5fxgha&p=strac&g=1

**Comment by lpgoldstein** [comment]
get mail to link to work

**Comment by mcharg** [comment]
@cthomasONRR The document is added. Just waiting on email icon for this one.

**Comment by mcharg** [comment]
@cthomasONRR Ready for ODDD approval. Created separate issue for icons. Already have SME approval.

**Comment by cthomasONRR** [comment]
@mcharg approved
